backend/echelon.py

```diff
@@ -0,0 +1,41 @@
+matrix=[]
+l2=[]
+m=int(input("enter number of rows >>>  \n"))
+n=int(input("enter number of columns >>>  \n"))
+for i in range(0,m):
+    matrix.append([])
+    for j in range(0,n):
+        matrix[i].append(0)
+for i in range (0,m):
```

Comment:
`range(0, m)` is the same as `range(m)`.
So, use the latter. That's one less thing to type :)